conans/client/installer.py

```diff
@@ -547,7 +549,8 @@ def _call_package_info(self, conanfile, package_folder, ref):
         conanfile.cpp_info.public_deps = public_deps
         # Once the node is build, execute package info, so it has access to the
         # package folder and artifacts
-        with pythonpath(conanfile):  # Minimal pythonpath, not the whole context, make it 50% slower
+        conan_v2 = get_env(CONAN_V2_MODE_ENVVAR, False)
```

Comment:
Better to define a different ``pythonpath`` context as ``no_op``, in its definition, instead of everywhere it is called.

Reply:
Q: **do we want to deprecate the [`pythonpath` tool](https://docs.conan.io/en/latest/reference/tools.html?highlight=pythonpath#tools-pythonpath)?** If yes, I'll add the CONAN_V2 check inside and it will raise for any usage; if not I will group this logic somewhere else.

Reply:
Probably we want to manage it together or similar to the environments. So at the moment we don't have a clear replacement. I'd prefer to just remove the internal automatic application of ``pythonpath()`` context, and replace it with a no-op for CONAN_V2_MODE

Reply:
So, yes, it is good at it, simpler than the alternative if we want to leave the users ``pythonpath`` untouched. Lets merge this.